src/pages/connect.tsx

```diff
@@ -280,7 +279,8 @@ const ConnectLanding: ThirdwebNextPage = () => {
             trackingCategory={TRACKING_CATEGORY}
             ctaLink="https://portal.thirdweb.com/payments"
             gradient="linear(to-r, #4490FF, #4490FF)"
-            lottie={checkoutLottie}
```

Comment:
we removed these, did we also delete the files? if not let's do that if they are no longer used

Reply:
Yeah we use that in the checkout landing page [here](https://github.com/thirdweb-dev/dashboard/blob/main/src/pages/checkout.tsx#L14)